libsoftwarecontainer/src/gateway/devicenodegateway.h

```diff
@@ -60,7 +60,7 @@
  *                        "mode":  "666"
  *                    }
  *                ]
- *  }
+ *
```

Comment:
jsonlint.com - paste the string and use the format the page turns it into, would make all y'all consistent